homeassistant/components/lcn/services.yaml

```diff
@@ -2,8 +2,11 @@
 
 output_abs:
   fields:
+    device_id:
+      example: "91aa039a2fb6e0b9f9ec7eb219a6b7d2"
+      selector:
+        text:
```

Comment:
So what is a device_id in this context? A HA device ID? Is the device we reference here a HA device?

Reply:
Yes, it is the HA device_id.

Reply:
Changed this to a device selector. Well, we have plenty of device models which makes the filter somewhat lengthy.